coders/dcm.c

```diff
@@ -4097,7 +4095,24 @@ static Image *ReadDCMImage(const ImageInfo *image_info,ExceptionInfo *exception)
           }
         option=GetImageOption(image_info,"dcm:rescale");
         if (option != (char *) NULL)
-          info.rescale=IsStringTrue(option);
+          info.rescale=(int) ParseCommandOption(MagickBooleanOptions,
+	    MagickFalse,option);
+        option=GetImageOption(image_info,"dcm:window");
+        if (option != (char *) NULL)
```

Comment:
I wonder if we should make it so that if this option is not null we set info.rescale to MagickTrue. This option will only do something when info.rescale is true. And the you would only need to do this: `convert -define dcm:window=2500x3000 MRIm5.dcm x.tif`